frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -256,7 +257,29 @@ function GridCardGraph({
 					onView={handleOnView}
 					onDelete={handleOnDelete}
 				/>
-				<Spinner height="20vh" tip="Loading..." />
+				{prevChartDataSetRef.current === undefined && (
+					<Spinner
+						height="12vh"
+						tip="Loading..."
+						customStyle={{
+							justifyContent: 'flex-end',
+							marginRight: '5px',
+						}}
+					/>
+				)}
+				{!isEmpty(widget) && prevChartDataSetRef.current && (
+					<GridGraphComponent
+						{...{
```

Comment:
please don't create a new object one to map mapping is fine

Reply:
@palashgdev - Can you explain me this in detail?